HANDBOOK.md

```diff
@@ -101,12 +101,13 @@ It is your choice to build the barriers first or the map first. For this example
   ```
   //mtschemplace map_terrain
   ```
-* Your terrain should be placed there. If you're not happy with the result, you can go back down, pick another terrain, and do it again.
+* You should place your terrain there. If you're unhappy with the result, you can return, pick another landscape, and do it again.
```

Comment:
```suggestion
* With that, your terrain should be placed there successfully. If you're unhappy with the result, you can go back down, pick another terrain, and do it again.
```
Since it is already placed at this point, I don't think this makes sense.
Terrain is used previously, so why are we using landscape now?
"Return" seems a bit vague, but there are probably better options.